.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for Trino Python client
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+### Describe the feature
+A clear and concise description of what you want to happen.
+
+### Describe alternatives you've considered
+A clear and concise description of any alternative solutions or features you've considered.
+
+### Additional context
+Please include any other relevant context here.
+
+### Who will this benefit?
+What kind of use case will this feature be useful for? Please be specific and provide examples, this will help us prioritize properly.
```

Comment:
The Trino python client is an open source project and there is no JIRA or task management involved - so there are no priorities. Things will get done when they'll get done and depending on how much effort someone puts in for a certain thing to get done.

cc: @martint

Reply:
While there may be no formal project management in place, there is some prioritization happening.  A bug leading to incorrect results will get more attention than a cosmetic issue.  Same thing with a feature that has a wider audience vs. a narrow use case.  An issue may be "prioritized" by virtue of getting more contributors interested in implementing it, because of its potential impact.

Reply:
Every project, including open source has priorities, bugs vs new features at least.
I don't see any issues with this sentence.

Reply:
This is about managing expectations. I'd avoid suggesting that any reported issues will have any priority. For non-native English speakers, this might be confusing, even to the point they may think that all issues have _some_ priority, where usually all of them are handled with best effort.

tl;dr let's avoid building up expectations we can't meet.